.travis.yml

```
@@ -1,25 +1,22 @@
 dist: xenial
 language: node_js
-install: npm ci
+install: yarn
 
 cache:
   directories:
     - node_modules
 
 jobs:
   include:
-      # Lint doesn't need to build
     - stage: lint
-      script: npm run lint
+      script: yarn lint
       node_js: "12"
     - stage: unit tests
-      # Test already builds
-      script: npm run test
+      script: yart test
       node_js: "10"
     - node_js: "12"
```

The test job fails with NodeJS 10 because better-discord 12 requires NodeJS ≥12.

```
error better-discord.js@12.3.1: The engine "node" is incompatible with this module. Expected version ">=12.0.0". Got "10.23.0"
```

The new release might be the occasion to drop compatibility with this old version of Node.

```suggestion
      node_js: "12"
```

__The readme file needs to be updated accordingly as well.__